src/Tizen.NUI/src/public/Utility/Capture.cs

```diff
@@ -333,12 +378,39 @@ public NativeImageSource GetNativeImageSource()
             Tizen.Log.Debug("NUI", $"GetNativeImageSource()");
             return new NativeImageSource(Interop.Capture.GetNativeImageSourcePtr(SwigCPtr), true);
         }
+
+        /// <summary>
+        /// Dispose
+        /// </summary>
+        /// <param name="type"></param>
```

Comment:
Please add a description about param name even though it's very brief.
```suggestion
        /// <param name="type">DisposeTypes</param>
```

Reply:
Thank you.